bin/install-amp-validator-test-data.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+set -e
+#
+# WP-AMP Validator Tests Script
+#
+# Note: This script assumes you have the following installed:
+#  - wp-cli
+#
+# from within the plugin root director enter 'bash bin/install-amp-validator-test-data.sh' to run the installer
+#####################################################################################
+
+# Check if our test data exists. If it already does not, we will get it from github.
+if ! [ -f "wptest.xml" ]
+then
+
+	# Get WP Test data if a newer file exists.
+	cd tests/assets && { curl -OL -z wptest.xml https://raw.githubusercontent.com/manovotny/wptest/master/wptest.xml; cd - ; }
+
+fi
+
+if [ "${TRAVIS}" = "true" ]; then
```

Comment:
We have to install WP-CLI on Travis every time. I also wanted to give the local user the option of running only one or none of the data imports and of noting being forced to have their URL structure rewritten.